src/services/dumpers/templates/agent-nodejs/docker-compose.hbs

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+{{#if network}}
+      network: {{network}}
+{{/if}}
+    container_name: {{ containerName }}
+    environment:
+      - FOREST_ENV_SECRET=${FOREST_ENV_SECRET}
+      - FOREST_AUTH_SECRET=${FOREST_AUTH_SECRET}
+{{#if forestServerUrl }}
+      - FOREST_SERVER_URL={{ forestServerUrl }}
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
+{{/if}}
+      - APPLICATION_PORT=${APPLICATION_PORT}
+      - DATABASE_URL={{ dbUrl }}
+      - DATABASE_SSL=${DATABASE_SSL}
+      - DATABASE_REJECT_UNAUTHORIZED=${DATABASE_REJECT_UNAUTHORIZED}
+{{#if dbSchema }}
+      - DATABASE_SCHEMA=${DATABASE_SCHEMA}
+{{/if}}
+      - NODE_ENV=${NODE_ENV}
+{{#if forestExtraHost}}
+    extra_hosts:
+      - {{forestExtraHost}}:host-gateway
+{{/if}}
+    ports:
+      - "${APPLICATION_PORT}:${APPLICATION_PORT}"
+    volumes:
+      - ./:/usr/src/app
+      - /usr/src/app/node_modules
```

Comment:
Isn't the image already built with `node_modules` embedded?

Reply:
👍